guides/common/modules/proc_managing-ostree-content-with-content-views.adoc

```diff
@@ -0,0 +1,53 @@
+[id="managing-ostree-content-with-content-views_{context}"]
+= Managing OSTree Content with Content Views
+
+Use Content Views to manage OSTree branches across the application life cycle.
+This process uses the same publication and promotion method as RPMs or Puppet modules.
+
+.Prerequisite
+* You have created a Product and added an OSTree repository to it.
+
+.Procedure
+. In the {Project} web UI, navigate to *Content* > *Content Views* and click *Create New View*.
+. In the *Name* field, enter a plain text name for the view.
+This automatically populates the *Label* field.
+. In the *Description* field, enter a description of the OSTree Content View.
+. If you want to use a Composite Content View, select the *Composite View* check box.
+. Optional: If you want to solve dependencies automatically every time you publish this Content View, select the *Solve Dependencies* check box. Dependency solving slows the publishing time and might ignore any Content View filters you use.
+. Optional: If this Content View is for importing from an upstream server only, select the *Import only* check box. You cannot publish import-only Content Views directly.
+. Click *Save* to complete.
+. Navigate to the *Repositories* tab.
+. Select the OSTree repository that you want to use.
+Click *Add Repository* to add the OSTree content from this repository to the Content View.
+. Click *Publish new version*.
+. In the *Description* field, enter a description for the version.
+. Optional: Enable the *Promote* switch to promote this Content View across environments in the application life cycle.
+. Click *Next*.
+. Review details and click *Finish*.
+
+.CLI Procedure
+. Obtain a list of repository IDs:
++
+[options="nowrap" subs="+quotes"]
+----
+# hammer repository list --organization "_My_Organization_"
+----
+. Create the Content View and add the repository:
++
+[options="nowrap" subs="+quotes"]
+----
+# hammer content-view create \
+--name "_OSTree_" \
+--description "_Example Content View for the OSTree_" \
+--repository-ids _5_ \
+--organization "_My_Organization_"
+----
+. Publish the view:
```

Comment:
```suggestion
. Publish the Content View:
```

We have mixed capitalizations on the master branch. @melcorr's call.

Reply:
It should always be Content View. I might ask @AkshayGadhaveRH to look for instances of this being wrong on master branch and correct it for us as an exercise. @AkshayGadhaveRH would you be willing to try this?

Reply:
@melcorr on it

Reply:
@AkshayGadhaveRH I'm going to correct this one and you can make another PR to fix it elsewhere. Agreed?

Reply:
Yes. Done in #1050 -rebase to enjoy